src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs

```diff
@@ -275,6 +275,27 @@ internal string GetString(int index, JsonTokenType expectedType)
             return lastString;
         }
 
+        internal DbRow GetRowAndSpan(int index, JsonTokenType expectedType, out ReadOnlySpan<byte> span)
+        {
+            CheckNotDisposed();
+            DbRow row = _parsedData.Get(index);
+
+            JsonTokenType tokenType = row.TokenType;
+
+            if (tokenType == JsonTokenType.Null)
+            {
```

Comment:
TODO: 
- [ ] find a way to cover this case.
- [ ] test to compare Null token type with a random text, throw or false?

Reply:
Currently the ValueEquals throws if the token type is not string, so this is unreachable.

FWIW: "span = null" is probably doing a weird sideways conversion through the array conversion operator; just use `span = default` or `span = ReadOnlySpan<byte>.Empty` (same thing).